src/imap/scan_folders.rs

```diff
@@ -79,16 +82,15 @@ impl Imap {
             }
         }
 
-        // We iterate over both folder meanings to make sure that if e.g. the "Sent" folder was deleted,
-        // `ConfiguredSentboxFolder` is set to `None`:
-        for config in &[
-            Config::ConfiguredSentboxFolder,
-            Config::ConfiguredSpamFolder,
-        ] {
-            context
-                .set_config(*config, folder_configs.get(config).map(|s| s.as_str()))
-                .await?;
-        }
+        // Make sure that if e.g. the "Sent" folder was deleted, `ConfiguredSentboxFolder` is set to `None`:
```

Comment:
```suggestion
        // Set the `ConfiguredSentboxFolder` or set it to `None` if the folder was deleted:
```
I think that's more understandable (IIRC I also wrote the previous comment, and I just needed several minutes to remember what it meant :joy:).